app/models/recipe_ingridient.rb

```diff
@@ -0,0 +1,4 @@
+class RecipeIngridient < ApplicationRecord
+  belongs_to :recipe
+  belongs_to :ingridient
```

Comment:
Here we can add `delegate`s that I mentioned earlier:

```
delegate :name, to: :ingridient
```

so we don't need to call:

```
recipe_ingridient.ingridient.name
```

but instead:

```
recipe_ingridient.name
```

the same thing for `unit_type`.